ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/NewWallet.hs

```diff
@@ -128,13 +166,35 @@ initNewWallet = do
   layout <- QVBoxLayout.new
   QWidget.setLayout newWallet layout
   QBoxLayout.setSpacing layout 18
-  QLayout.setContentsMarginsRaw layout 24 42 24 24
+  QLayout.setContentsMarginsRaw layout 24 24 24 24
+
+  modeSelector <- createModeSelector
+
+  QBoxLayout.addWidget layout modeSelector
+  void $ QLayout.setWidgetAlignment layout modeSelector $ alignHCenter .|. alignVCenter
+  QBoxLayout.addSpacing layout 42
 
   walletNameLabel <- QLabel.newWithText ("<b>WALLET NAME</b>" :: String)
   walletName <- QLineEdit.new
   QLineEdit.setPlaceholderText walletName ("wallet name" :: String)
 
   addRow layout walletNameLabel walletName
+
+  walletMnemonicLabel <- QLabel.newWithText ("<b>MNEMONIC PHRASE</b>" :: String)
+  walletMnemonic <- QLineEdit.new
+
+  (walletMnemonicWidget, walletMnemonicLayout) <- createSubWidget
+  addRow walletMnemonicLayout walletMnemonicLabel walletMnemonic
+
+  fullRestoreLabel <- QLabel.newWithText ("Perform full restore" :: String)
+  fullRestore <- QCheckBox.new
```

Comment:
I think it should be enabled by default (like in TUI).

Reply:
Done.